assign_rights/tests.py

```diff
@@ -219,14 +219,18 @@ def test_get_dates(self):
 
     def test_create_json(self):
         """Tests that Serialzers are working as expected."""
-        for obj_cls, serializer_cls in [
-                (RightsShell, RightsShellSerializer),
-                (RightsGranted, RightsGrantedSerializer)]:
+        for obj_cls in [RightsShell, RightsGranted]:
             obj = random.choice(obj_cls.objects.all())
```

Comment:
Just noticing this - since the RightsShells now have different serializers based on the rights_basis, I think we'll want  to make sure we test with a RightsShell from each of the four rights bases.

Reply:
I'm looking at this now, and am wondering if we need to do a different setup for adding rights shells to make sure that we get at least one of each basis, or add additional shells as part of this method. Does that sound right?

Reply:
Yes I think so. I would think doing it in the setup probably makes more sense, as that gets used across multiple tests and will make everything better?